src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -124,15 +132,31 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
                 />
               </div>
               <div className="profile-text">
-                <h2 className="italic">
-                  {selectedUserProfile.partyName || DEFAULT_PARTY_NAME}
-                </h2>
+                <h2 className="italic">{partyName || DEFAULT_PARTY_NAME}</h2>
               </div>
             </div>
             <div className="profile-extras">
               {renderedProfileQuestionAnswers}
             </div>
             <div>{renderedProfileLinks}</div>
+            {realName && (
+              <>
+                <p className="light no-margin">Full Name</p>
+                <h6>{realName}</h6>
+              </>
+            )}
+            {companyDepartment && (
+              <>
+                <p className="light no-margin">Department</p>
+                <h6>{companyDepartment}</h6>
+              </>
+            )}
+            {companyTitle && (
+              <>
+                <p className="light no-margin">Title</p>
+                <h6>{companyTitle}</h6>
+              </>
+            )}
```

Comment:
Same "don't abuse `h6` for styling" comments as above. Bootstrap has a class that mirrors the heading styles I believe. At the very least use that.

Reply:
@0xdevalias note that this is targeted to `GitHub` and probably we won't use these fields anywhere except this

Reply:
I can't find what you are referring to here either
https://hackerthemes.com/bootstrap-cheatsheet/

Reply:
See link + explicit references above in https://github.com/sparkletown/sparkle/pull/1667#discussion_r659707405

Reply:
Done in https://github.com/sparkletown/sparkle/commit/41dc9b23e005d6755989c6684484c7568675d1d9